recipes/flex/all/conanfile.py

```diff
@@ -82,6 +82,8 @@ def package(self):
     def package_info(self):
         self.cpp_info.libs = ["fl"]
         self.cpp_info.system_libs = ["m"]
+        # Avoid CMakeDeps messing with Conan targets
+        self.cpp_info.set_property("cmake_find_mode", "none")
```

Comment:
I completely missed that 😮‍💨

Reply:
👋 

Why? Now a build using Conan requires the modules provided by CMake ([this `FLEX_INCLUDE_DIRS`](https://gitlab.kitware.com/cmake/cmake/blob/master/Modules/FindFLEX.cmake#L122)) to find the `include` folder provided by the Conan package.

I think it's ok to remove the targets (official one doesn't provide targets) but a way to populate just the same variables as `FindFLEX` would be nice (not the first time I've been here: https://github.com/conan-io/conan-center-index/issues/1813). 

Do we prefer to rely on CMake files or to provide our own (maybe some hidden/wrong-name target) with a `build_modules` to provide the extra macros, vars,...